src/js/components/TaskTable.js

```diff
@@ -196,7 +189,10 @@ class TaskTable extends React.Component {
 
 TaskTable.propTypes = {
   className: React.PropTypes.string,
-  parentRouter: React.PropTypes.func.isRequired,
+  parentRouter: React.PropTypes.oneOfType([
```

Comment:
Why do we need to pass down the `ServiceDetailTaskTab` router as a prop to all these components? Is it a workaround for tests?